### PR TITLE
Dataupload

### PR DESCRIPTION
**Pull request by edignot**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- 

## What is the new behavior?

- UserInfo and Profile tests completed
- Image upload setup updated
- Some user info card styling fixed

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: code/web/src/modules/user/UserInfo.js [hunk 1]
```diff
@@ -2,251 +2,248 @@
...
 
-  toggleEdit = () => {
-    this.setState({ isEditing: !this.state.isEditing });
-  };
-
-  handleChange = (e) => {
-    const { name, value } = e.target;
-    this.setState({ [name]: value });
-  };
+    imageSelectedHandler = (e) => {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:43]: Thanks for getting that ready

File: code/web/src/modules/user/UserInfo.test.js [hunk 2]
```diff
@@ -37,100 +37,140 @@ describe('<UserInfo/>', () => {
...
         const { getByText, getByTestId } = UserInfoContainer;
         fireEvent.click(getByText('Edit'));
         expect(getByTestId('shipping-input').value).toBe(
             'Add your shipping address'
         );
     });
-    test('Shipping address can be changed when in edit mode', () => {
-        const { getByText, getByTestId } = UserInfoContainer;
+
+    test('User info form inputs can be edited when in editing mode', () => {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.test.js:135]: Good job pulling these in from the Profile.test.js

File: code/web/src/modules/user/api/actions.js [hunk 3]
```diff
@@ -1,119 +1,129 @@
...
 // Actions
 
 // Set a user after login or using localStorage token
 export function setUser(token, user) {
-  if (token) {
-    axios.defaults.headers.common['Authorization'] = `Bearer ${token}`;
-  } else {
-    delete axios.defaults.headers.common['Authorization'];
-  }
+    if (token) {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/api/actions.js:19]: 4 space indent?

Comment (edignot) [reply code/web/src/modules/user/api/actions.js:19]: sorry ...

File: code/web/src/modules/user/Profile.test.js [hunk 4]
```diff
@@ -1,127 +1,52 @@
-import React from 'react'
-import Profile from './Profile'
+import React from 'react';
+import Profile from './Profile';
 
-import { render, fireEvent } from '@testing-library/react'
-import { BrowserRouter } from  'react-router-dom'
-import '@testing-library/jest-dom'
+import { render } from '@testing-library/react';
```
Comment (sagemlee) [inline code/web/src/modules/user/Profile.test.js:4]: What is a "fireEvent"? Sounds ominous.

File: code/web/src/modules/user/Profile.test.js [hunk 5]
```diff
@@ -1,127 +1,52 @@
...
+    });
+
+    test('Profile header is displayed correctly ', () => {
+        const { getByText } = profileContainer;
+        expect(getByText('My profile')).toBeInTheDocument();
+    });
+
+    test('User info card is displayed correctly ', () => {
+        const { getByText } = profileContainer;
+        expect(getByText('About me')).toBeInTheDocument();
```
Comment (sagemlee) [inline code/web/src/modules/user/Profile.test.js:45]: We have a similar command for our rails testing called "expect page to have"

File: code/api/package.json [hunk 6]
```diff
@@ -1,49 +1,49 @@
...
+        "express-graphql": "0.9.0",
+        "graphql": "14.6.0",
+        "jsonwebtoken": "8.5.1",
+        "morgan": "1.10.0",
+        "multer": "1.4.2",
+        "mysql2": "2.1.0",
+        "pg": "^8.2.1",
+        "sequelize": "5.21.6",
+        "sequelize-cli": "^5.5.1"
+    }
```
Comment (arpariseau) [inline code/api/package.json:48]: so uh, did anything change here aside from the indentations?

Comment (edignot) [reply code/api/package.json:48]: no, sorry, I have setting for 2 now, but it doesn't change it back

**Pull request merged**